<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductManagerForm</class>
 <widget class="QWidget" name="ProductManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>377</width>
         <height>306</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="productFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="productIdLabel">
            <property name="text">
             <string>&amp;ID</string>
            </property>
            <property name="buddy">
             <cstring>productIdLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="productIdLineEdit">
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="productNameLabel">
            <property name="text">
             <string>&amp;Name</string>
            </property>
            <property name="buddy">
             <cstring>productNameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="productNameLineEdit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="priceLabel">
            <property name="text">
             <string>Price</string>
            </property>
            <property name="buddy">
             <cstring>priceLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="priceLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="stockLabel">
            <property name="text">
             <string>Stock</string>
            </property>
            <property name="buddy">
             <cstring>stockLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="stockLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="amHorizontalLayout">
          <item>
           <widget class="QPushButton" name="addPushButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyPushButton">
            <property name="text">
             <string>Modify</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearPushButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>377</width>
         <height>306</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="idHorizontalLayout">
          <item>
           <widget class="QComboBox" name="searchComboBox">
            <item>
             <property name="text">
              <string>ID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Price</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Stock</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="searchPushButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="searchTreeWidget">
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Price</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Stock</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stock</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
